private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -678,13 +1154,14 @@ export class TemplateApi {
         });
     }
     /**
-     * Update existing template given id
-     * @summary Update existing template
-     * @param templateId ID of template to update
-     * @param body Template object to be added
+     * Returns a single template
+     * @summary Find template by id
+     * @param templateId ID of template to return
+     * @param isPublished Query based on if template is published
+     * @param version Version of template to return
      * @param {*} [options] Override http request options.
      */
-    public templateTemplateIdPost (templateId: string, body: TemplateJSON, options: any = {}) : Promise<{ response: http.IncomingMessage; body?: any;  }> {
+    public templateById (templateId: string, isPublished?: boolean, version?: string, options: any = {}) : Promise<{ response: http.IncomingMessage; body: TemplateList;  }> {
```

Comment:
I might be missing something here, but why would we want to get a templateByID and provide a param for isPublished? If we want a template by ID then I don't think we need isPublished

Reply:
It's for if you want to get only the published/unpublished versions of a template

Reply:
ohhh okay cool